smarts/core/sumo_road_network.py

```diff
@@ -231,8 +231,11 @@ def _compute_road_polygons(self):
                 lane_to_poly[lane.getID()] = shape
 
         # Remove holes created at tight junctions due to crude map geometry
+        # NOTE: If you don't want snapping of a particular edge, set its type to "ignore_snapping" in
+        # the scenario's map.net.xml file. See scnenario/ngsim/map.net.xml for example.
```

Comment:
I am curious, is this inbuilt or a custom type?

Reply:
No, obviously custom type.